e2e/vaultsecrets/input/default_wi.nomad.hcl

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+job "default_wi" {
+  datacenters = ["dc1", "dc2"]
```

Comment:
Because of the default wildcard, we can omit this now:

```suggestion
```

Reply:
I always forget this. Updated within Removed the comment within https://github.com/hashicorp/nomad/pull/25139/commits/c3ef5f1b98720cc24ebd5e8edb494a8c4ecd54e4